Telerik.Examples.RazorPages/Telerik.Examples.RazorPages/Pages/Captcha/CaptchaIndex.cshtml.cs

```diff
@@ -0,0 +1,108 @@
+using Microsoft.AspNetCore.Mvc;
+using Telerik.Examples.RazorPages.Models;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.AspNetCore.Hosting;
+using System.IO;
+using Telerik.Web.Captcha;
+using Microsoft.AspNetCore.Http;
+using System.Drawing.Imaging;
+
+namespace Telerik.Examples.RazorPages.Pages.Captcha
+{
+    public class CaptchaIndexModel : PageModel
+    {
+        [BindProperty]
+        public OrderViewModel Order { get; set; }
+
+        protected readonly IWebHostEnvironment HostingEnvironment;
+        protected string CaptchaPath { get; set; }
+
+        public CaptchaIndexModel(IWebHostEnvironment hostingEnvironment)
+        {
+            HostingEnvironment = hostingEnvironment;
+
+            var captchaFilePath = Path.Combine("shared", "UserFiles", "Folders", "Captcha");
+            CaptchaPath = $"{HostingEnvironment.WebRootPath}\\{captchaFilePath}";
+        }
+
+        public void OnGet(CaptchaModel captchaModel)
+        {
+            if (string.IsNullOrEmpty(captchaModel.CaptchaID))
+            {
+                GenerateNewCaptcha();
+            }
+            else
+            {
+                string text = GetCaptchaText(captchaModel.CaptchaID);
+
+                if (text == captchaModel.Captcha.ToUpperInvariant())
+                {
+                    ModelState.Clear();
+                    GenerateNewCaptcha();
+                }
+            }
+            if (Order == null)
+            {
+                Order = new OrderViewModel();
+            }
+        }
+        public IActionResult OnPost()
+        {
+            if (!ModelState.IsValid)
+            {
+                return Page();
+            }
+
+            return RedirectToPage("/Form/Success");
```

Comment:
When you click "Go back" in the "Success" page, the button navigates you to "FormStandardSubmit" example.